python/lsst/summit/utils/guiders/reading.py

```diff
@@ -689,47 +700,47 @@ def get(
         guiderData : `GuiderData`
             Assembled guider data object.
         """
-        # check if the guider name is swaped (dayObs < 20250509)
+        # Check if the guider name is swapped (dayObs < 20250509)
         # modifies self.guiderNameMap in place if necessary
         self.applyGuiderNameSwapIfNeeded(dayObs)
 
-        # 1. Get guider_raw stamps all guiders
+        # 1. Get guider_raw stamps for all guiders (never modified)
         rawStampsDict = self.getGuiderRawStamps(dayObs, seqNum)
 
-        # set the number of stamps
-        # determine the maximum number of stamps among all guiders
+        # Determine the maximum number of stamps among all guiders
         nStampsList = [len(stamps) for stamps in rawStampsDict.values()]
-        self.nStamps = max(nStampsList)
+        nStamps = max(nStampsList)
 
-        if self.nStamps <= 1:
+        if nStamps <= 1:
             raise RuntimeError(
-                f"Only {self.nStamps} stamps found for dayObs {dayObs}, seqNum {seqNum}. "
+                f"Only {nStamps} stamps found for dayObs {dayObs}, seqNum {seqNum}. "
                 "At least 2 stamps are required to create GuiderData."
             )
 
-        # Create a visitinfo dataId to get the WCS and camera rotation angle
+        # 2. Get WCS map and camera rotation angle
         visitInfo = getVisitInfo(self.butler, dayObs, seqNum, scienceDetNum)
-
-        # 2. Get WCS map for all guiders
         wcsMapDict = makeInitGuiderWcs(self.camera, visitInfo)
-
-        # 3. Get camera rotation angle
```

Comment:
I don't think these mattered much in the first place, but renumbering them for no reason is also not great practice (probably the AI going a little rogue - they like to do random things like that sometimes).

Reply:
Done.